js/main.js

```diff
@@ -0,0 +1,75 @@
+const speaker = [
+  {
+    name: 'Mousa YAYA',
+    title: 'CEO and Founder of global school',
+    image: 'images/speaker1.png',
+    biblio: 'Created Global school in 2012 to bring all the student together across the world widthout discrimination',
+  },
+  {
+    name: 'Richard YONON',
+    title: 'Professor of sociology at Harvard University',
+    image: 'images/speaker2.png',
+    biblio: 'Richard is a great sociologue, he published his seminal book "Nature and Human geing" in 1995',
+  },
+  {
+    name: 'Toyi Manha',
+    title: 'Ministre of youth in Ghana',
+    image: 'images/speaker3.png',
+    biblio: 'Toyi works for potential changes and hope for the youth in the wholeword specialy in africa',
+  },
+  {
+    name: 'Julia LEDA',
+    title: 'Professor of sociology at Harvard University',
+    image: 'images/speaker4.png',
+    biblio: 'Julia is a great sociologue, he published his seminal book "Nature and Human geing" in 1995',
+  },
+  {
+    name: 'Mousa YAYA',
+    title: 'CEO and Founder of global school',
+    image: 'images/speaker5.png',
+    biblio: 'Created Global school in 2012 to bring all the student together across the world widthout discrimination',
+  },
+  {
+    name: 'Richard YONON',
+    title: 'Professor of sociology at Harvard University',
+    image: 'images/speaker6.png',
+    biblio: 'Richard is a great sociologue, he published his seminal book "Nature and Human geing" in 1995',
+  },
+];
+const containSpeaker = document.querySelector('.contain_speaker');
+
+function ftnSpeaker(n) {
+  const cardSpeaker = `
+      <div class="speaker">
+        <img src="${speaker[n].image}" alt="speaker's peature">
+      <div>
+      <h3 class="name">${speaker[n].name}</h3>
+      <h4>${speaker[n].title}.</h4>
+      <p>${speaker[n].biblio}</p>
+      </div>
+      </div>`;
```

Comment:
- Great job for making the speaker list dynamic 👍, I think it would be better if you wrap each speaker with `article` for semantic reasons.
- Kindly, use the same spacing and background image for speaker images to make the page more user-friendly for desktop and mobile versions.
### Given Design 👇
![2023-04-25_13-28_1](https://user-images.githubusercontent.com/52010789/234258963-bc74a089-186f-41da-8b3f-40dc3fd62da9.png)
### Your Design 👇
![2023-04-25_13-30](https://user-images.githubusercontent.com/52010789/234258976-e1092f16-8367-4e0d-aa1a-21a8a61e6462.png)